README.md

```diff
@@ -1 +1,976 @@
+name:
+```
+$ BALENARC_DATA_DIRECTORY=. balena device move 94095f8 --application altApp
+94095f8 was moved to altApp
+```
+A call of `balena device` specifying the UUID of the moved device will now
+show it is owned by the specified application:
+```
+$ BALENARC_DATA_DIRECTORY=. balena device 7654321
+== WEATHERED WILDFLOWER
+ID:                 1693707
+DEVICE TYPE:        fincm3
+STATUS:             idle
+IS ONLINE:          true
+IP ADDRESS:         192.168.1.171 192.168.1.169
+APPLICATION NAME:   altApp
+UUID:               76543217654321765432176543217654
+COMMIT:             7efbc95825641b6482742a54c8e74010
+SUPERVISOR VERSION: 9.15.7
+IS WEB ACCESSIBLE:  false
+OS VERSION:         balenaOS 2.38.0+rev1
+DASHBOARD URL:      https://dashboard.balena-cloud.com/devices/76543217654321765432176543217654/summary
+```
+The Supervisor on the device will remove any previously running services, as
+well as their images and any volumes associated with them and download the
+images associated with the new application before starting them.
+
+## 2.2 Moving Devices between Environments
+
+Moving a device between balena environments is slightly more involved,
+and alters depending on whether you're using a device running a development
+or production image.
+
+### 2.2.1 Development Devices
+
+For a device running a development image, you can use `balena leave` and
+`balena join` to carry this out.
+
+First, provision a device using a development image. You can do this using
+the balenaCloud dashboard and downloading a 'Development' edition image from
+the `cliApp` application page. Provision your balenaFin with this image
+using either balenaEtcher or balenaCLI.
+
+Once the device is provisioned and has connected to the balena network,
+discover its hostname or IP address by using `balena devices`:
+```
+$ BALENARC_DATA_DIRECTORY=. balena devices
+ID      UUID    DEVICE NAME      DEVICE TYPE     APPLICATION NAME STATUS IS ONLINE SUPERVISOR VERSION OS VERSION           DASHBOARD URL
+1234567 7654321 little-paper     fincm3          cliApp           Idle   true      9.15.7             balenaOS 2.38.0+rev1 https://dashboard.balena-cloud.com/devices/76543217654321765432176543217654/summary
+
+$ BALENARC_DATA_DIRECTORY=. balena device 7654321
+== LITTLE PAPER
+ID:                 1234567
+DEVICE TYPE:        fincm3
+STATUS:             idle
+IS ONLINE:          true
+IP ADDRESS:         192.168.1.171
+APPLICATION NAME:   cliApp
+UUID:               76543217654321765432176543217654
+COMMIT:             7efbc95825641b6482742a54c8e74010
+SUPERVISOR VERSION: 9.15.7
+IS WEB ACCESSIBLE:  false
+OS VERSION:         balenaOS 2.38.0+rev1
+DASHBOARD URL:      https://dashboard.balena-cloud.com/devices/76543217654321765432176543217654/summary
+```
+As we now have the local IP address for it, we can use this to call the command
+to leave the balenaCloud environment:
+```
+$ BALENARC_DATA_DIRECTORY=. balena leave 192.168.1.171
```

Comment:
Not important, but I tried this with a NUC VM running via the VBox NAT networking stack and the command just hung.

Reply:
Interesting, tried this on both a Pi4 and Fin and it worked. What did the Supervisor on the VM say?